tiledb/sm/array/array_directory.cc

```diff
@@ -1313,13 +1315,26 @@ bool ArrayDirectory::consolidation_with_timestamps_supported(
 
 shared_ptr<const Enumeration> ArrayDirectory::load_enumeration(
     const std::string& enumeration_path,
-    const EncryptionKey& encryption_key) const {
+    const EncryptionKey& encryption_key,
+    MemoryTracker& memory_tracker) const {
   auto timer_se = resources_.get().stats().start_timer("sm_load_enumeration");
 
   auto enmr_uri = uri_.join_path(constants::array_schema_dir_name)
                       .join_path(constants::array_enumerations_dir_name)
                       .join_path(enumeration_path);
 
+  uint64_t size = 0;
```

Comment:
We sure can!

Though for others, we chatted real quick OOB and went through a couple permutations of passing the MemoryTracker to `GenericTileIO::load` then passing the file size read instead of the tracker to avoid the duplicate file sizes. And then we figured out that the file size isn't correct anyway since its the compressed size. So now we're back to using tile.size() instead and future work will be required to preemptively check whether we can load before reading.